image-layout.md

```diff
@@ -21,7 +21,7 @@ It also contains a file that is used to identify the layout version:
 
 This is an example image layout:
 
-```
+```shell
```

Comment:
`console` is the right directive.

@wking I'd recommend at least looking at the github markdown docs before giving direction on a PR.